glide.lock

```diff
@@ -271,6 +271,7 @@ imports:
   - kubernetes/typed/storage/v1/fake
   - kubernetes/typed/storage/v1beta1
   - kubernetes/typed/storage/v1beta1/fake
+  - pkg/api/v1
```

Comment:
@msau42 & @wongma7 - For this PR I'll plan to copy in the one const, and get rid of the vendor updates.